projects/addon-editor/constants/tiptap-editor-styles.ts

```diff
@@ -288,4 +288,36 @@ export const tiptapEditorStyles = `
    transform: rotate(225deg);
    top: 1.75rem;
 }
+
+.t-editor-placeholder::before {
+  content: attr(data-placeholder);
+  float: left;
+  color: #adb5bd;
```

Comment:
can we choose a color from our pallet? to support night theme

Reply:
fixed